src/main/resources/config/application.yml

```diff
@@ -45,6 +45,11 @@ spring:
             enabled: false
     thymeleaf:
         mode: XHTML
+    http:
+        multipart:
+            max-file-size: 10MB
+            max-request-size: 10MB
+
```

Comment:
This probably does exactly the same as I was wondering before :-) If yes, please add a comment in the File handler about these properties here.